datasets/trivia_qa/README.md

```diff
@@ -288,7 +288,8 @@ The data fields are the same among all splits.
 
 ### Licensing Information
 
-The TriviaQA dataset is released under the [Apache-2.0 License](http://www.apache.org/licenses/LICENSE-2.0).
+The TriviaQA dataset is released under the [Apache-2.0 License](http://www.apache.org/licenses/LICENSE-2.0). The University of Washington does not own the copyright of the questions and documents included in TriviaQA. For more information see [this](http://nlp.cs.washington.edu/triviaqa/).
```

Comment:
I don't see the mention of Apache 2.0 License anywhere but in the GH repo [here](https://github.com/mandarjoshi90/triviaqa), which contains the preprocessing/evaluation code and not the data, so let's delete the Apache 2.0 License part and the homepage link because it's already present in the card.

```suggestion
The University of Washington does not own the copyright of the questions and documents included in TriviaQA.
```